### DISCUSSION TITLE
server error when trying to initialise nextcloud for the first time

### DISCUSSION THREAD

**Discussion by amjarman** [nextcloud/all-in-one#3169] [Questions] [answered]
Hi, im getting a strange server error when trying to initialise nextcloud for the first time. 
Also it seems the Apach, Nextcloud & Push are not running but when I look in Portainer they show healthy. 
I have set up NC AIO many time but this is the first time I have done it on a windows PC and it been a nightmare. I have installed Ubuntu on the PC but this has not helped.
I also don't know how to view the error logs from the master container: Tips: To display error details in HTTP response set "displayErrorDetails" to true in the ErrorHandler constructor.
![image](https://github.com/nextcloud/all-in-one/assets/61504173/00f4a865-bd94-49d5-af0d-636c6ff370ff)

![nc](https://github.com/nextcloud/all-in-one/assets/61504173/a52c82a0-5304-4162-b742-b834ffb085f7)

_Originally posted by @amjarman in https://github.com/nextcloud/all-in-one/discussions/2105#discussioncomment-6734937_

**Comment by szaimen** [comment, marked as answer]
Hi, can you post the mastercontainer logs here please?
